Detail the default permissions for built-in roles (#719)
OctopusDeploy/docs: docs/administration/managing-users-and-teams/user-roles.md

The built-in user roles can be modified to contain more or less roles to suit specific needs. But instead of modifying the built-in ones, we recommend that you leave them as an example and instead create your own user roles.

:::success

> To view the default permissions for each of the built-in user roles, please see [default permissions](/docs/administration/managing-users-and-teams/default-permissions.md)

Reviewer: Period :)
```suggestion
To view the default permissions for each of the built-in user roles, please see [default permissions](/docs/administration/managing-users-and-teams/default-permissions.md).
```

| Space Manager        | Space managers can do everything within the context of the space they own. |

:::success

> For more information regarding the 'system or space level', please see [system and space permissions](/docs/administration/managing-users-and-teams/system-and-space-permissions.md)

Reviewer: Changed the emphasis from single quotes to italics and added a period.
```suggestion
For more information regarding the _system or space level_, please see [system and space permissions](/docs/administration/managing-users-and-teams/system-and-space-permissions.md).
```

Once the custom role is saved, the new role will be available to be assigned to teams in Octopus. [Some rules apply](/docs/administration/managing-users-and-teams/system-and-space-permissions.md#SystemAndSpacePermissions-RulesOfTheRoad), depending on the mix of system or space level permissions you chose.

> When applying roles to a team, you are able to optionally specify a scope for each role applied. This enables some complex scenarios, like granting a team [different levels of access](/docs/administration/managing-users-and-teams/creating-teams-for-a-user-with-mixed-environment-privileges.md) based on the environment they are authorized for.

Reviewer: ```suggestion
When applying roles to a team, you can optionally specify a scope for each role applied. This enables some complex scenarios, like granting a team [different levels of access](/docs/administration/managing-users-and-teams/creating-teams-for-a-user-with-mixed-environment-privileges.md) based on the environment they are authorized for.
```
